### ISSUE TITLE
P(q)*S(q) not doing the right thing (Trac #750)

### ISSUE THREAD

**Issue by ajj** [SasView/sasview#866]
(1) Problem with radius_effect appearing - name change on model is the cause.

(2) Extra scale appearing that shouldn't - coming from P(q), but isn't supposed to show up

(3) Background appears to be being multiplied by scale_factor

(4) Behaviour is wrong when P(q) nominally goes to 1 - getting 0 as result

Migrated from http://trac.sasview.org/ticket/750
```json
{
    "status": "closed",
    "changetime": "2016-11-03T09:33:19",
    "_ts": "2016-11-03 09:33:19.408832+00:00",
    "description": "(1) Problem with radius_effect appearing - name change on model is the cause.\n\n(2) Extra scale appearing that shouldn't - coming from P(q), but isn't supposed to show up\n\n(3) Background appears to be being multiplied by scale_factor\n\n(4) Behaviour is wrong when P(q) nominally goes to 1 - getting 0 as result",
    "reporter": "ajj",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2016-10-10T13:43:44",
    "component": "SasView",
    "summary": "P(q)*S(q) not doing the right thing",
    "priority": "blocker",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "ajj",
    "type": "defect"
}
```

**Comment by ajj** [comment]
Trac update at `2016/10/10 13:52:21`: **ajj** commented:

Looks like MultiplicationModel.py is doing:


```text
scale_factor * [P(q)+bkd]*S(q)
```

should be doing


```text
scale_factor*P(q)*S(q)+bkd
```

**Comment by pkienzle** [comment]
Trac update at `2016/10/10 15:28:42`: **pkienzle** commented:

sasmodels.product should be a replacement for !MultiplicationModel.  Incorporate that into sasview rather than fixing !MultiplicationModel, as per Ticket #680.

**Comment by ajj** [comment]
Trac update at `2016/10/10 15:52:43`: 
* **ajj** commented:

In changeset fd62331a2d3189e76b6046c10a411463ce7e9641:

```text
#!CommitTicketReference repository="sasview" revision="fd62331a2d3189e76b6046c10a411463ce7e9641"
Updating muliplicationmodel to be correct.

Fixes #866
```


* **ajj** changed resolution from "" to "fixed"
* **ajj** changed status from "new" to "closed"

**Comment by ajj** [comment]
Trac update at `2016/10/10 18:23:01`: **ajj** commented:

In changeset 68669da9154d53fe949cb741efa850cd516ce157:

```text
#!CommitTicketReference repository="sasview" revision="68669da9154d53fe949cb741efa850cd516ce157"
Updating MultiplicationModel to deal with results of fixing #788.

Fixes #866 (again)
```

**Comment by ajj** [comment]
Trac update at `2016/11/03 09:33:17`: **ajj** commented:

In changeset c5a22a8553ed868bd3d5fd17df0b9f823de4d30c:

```text
#!CommitTicketReference repository="sasview" revision="c5a22a8553ed868bd3d5fd17df0b9f823de4d30c"
Updating muliplicationmodel to be correct.

Fixes #866
```

**Comment by ajj** [comment]
Trac update at `2016/11/03 09:33:19`: **ajj** commented:

In changeset 736f27e9e612490fce2428fe33d9f38530134575:

```text
#!CommitTicketReference repository="sasview" revision="736f27e9e612490fce2428fe33d9f38530134575"
Updating MultiplicationModel to deal with results of fixing #788.

Fixes #866 (again)
```
